README.md

```diff
@@ -233,6 +244,19 @@ they exist in the local directory:
   support is enabled. All EFI binaries included in the image's ESP are
   signed with this key, as a late step in the build process.
 
+* `mkosi.output/` may be a directory. If it exists, and the image
```

Comment:
Hmm, I think description starts in the wrong place, because it doesn't take into account the case when `--output` is used. What about:

> An output directory may be specifed using the `--output` option, or if that option is not used and `mkosi.output/` directory exists, it will be used as the output directory automatically. If an output directory is used, all outputs will be written to this directory, unless their name is explicitly configured to a path that contains at least one slash (i.e. plain file names still end up in the output directory, but anything resembling a path is used as is). This applies to all build artifacts (the image itself, the root hash file in case Verity is used, the checksum and its signature if enabled, and the nspawn settings file if any). This setting is particularly useful if multiple different images shall be built from the same working directory, as otherwise the build result of a preceeding run might be copied into a build image as part of the
source tree (see above).

Reply:
> An output directory may be specifed using the --output option

But that's misleading. With --output= you specify the output image path, not the dir to place it in. Except when we do -t directory the result will be a file, not a directory after all.

Also, I'd really like to keep this in the list of .mkos-* files and directories, instead of moving this elsewhere.